src/evict/evict_lru.c

```diff
@@ -628,11 +628,10 @@ __evict_update_work(WT_SESSION_IMPL *session)
 	 * (3) the cache is more than half way from the dirty target to the
 	 *     dirty trigger.
 	 */
-	if (!F_ISSET(conn, WT_CONN_EVICTION_NO_LOOKASIDE) &&
-	    (__wt_cache_stuck(session) ||
```

Comment:
I don't understand how this change is related - and it makes me a bit apprehensive that we'll queue pages we wouldn't have queued before (maybe that won't happen?)

Reply:
This was fixing a bug: `rollback_to_stable` sets `WT_CONN_EVICTION_NO_LOOKASIDE` as its way to make sure no lookaside eviction happens, but only had an effect after the eviction ran and clearing `WT_CACHE_EVICT_LOOKASIDE` from the cache flags.

What I did was move the `WT_CONN_EVICTION_NO_LOOKASIDE` check into `__evict_review` so it will be checked immediately (or at least reliably after `__txn_rollback_eviction_drain` finishes).  I hope that makes sense: let me know if I can add comments to clarify.